core/src/main/scala/cats/syntax/apply.scala

```diff
@@ -23,13 +23,8 @@ package cats
 package syntax
 
 trait ApplySyntax extends TupleSemigroupalSyntax {
-  implicit final def catsSyntaxApply[F[_], A](fa: F[A])(implicit F: Apply[F]): Apply.Ops[F, A] =
-    new Apply.Ops[F, A] {
-      type TypeClassType = Apply[F]
-
-      val self = fa
-      val typeClassInstance = F
-    }
+  implicit final def applySyntaxBinCompat1[F[_], A](fa: F[A]): ApplySyntaxBinCompat1[F, A] =
```

Comment:
We should follow the naming convention `catsSyntaxApply...` to avoid name clashes.

```suggestion
  implicit final def applySyntaxBinCompat1[F[_], A](fa: F[A]): ApplySyntaxBinCompat1[F, A] =
```